paper.tex

```diff
@@ -1964,6 +1968,12 @@ \section{Conclusion and Future Work}
 focused only on the features that made it into the final app from the start we
 could have had more time to polish and refine those features.
 
+On a similar note, the requirements and scope of the project changed greatly
+from week to week, and sometimes even within one day. We spent a lot of time
```

Comment:
sort of don't like the term "a lot" but i could go either way

Reply:
Replaced with "significant amount of time"